concrete/src/Page/Cloner.php

```diff
@@ -146,6 +147,7 @@ public function clonePage(Page $page, ClonerOptions $options, Page $newParentPag
         $newPage->update($args);
 
         Section::registerDuplicate($newPage, $page);
+        $this->duplicateExpressEntitiesOnPageDuplicate($newPage, $options);
```

Comment:
When we clone a page if **option duplicate express entity** is active ,we launch method that  duplicate all **aliased** express form block found  in a page and create new collection version.